src/prefect/run_configs/kubernetes.py

```diff
@@ -108,6 +121,16 @@ def __init__(
         if image_pull_secrets is not None:
             image_pull_secrets = list(image_pull_secrets)
 
+        image_pull_policies = {"Always", "IfNotPresent", "Never"}
+        if (
+            image_pull_policy is not None
+            and image_pull_policy not in image_pull_policies
+        ):
+            raise ValueError(
+                f"Invalid image_pull_policy {image_pull_policy}.  "
```

Comment:
```suggestion
                f"Invalid image_pull_policy {image_pull_policy!r}.  "
```

For displaying string values like this, it's nice to get the repr which will put quotes so

> Invalid image_pull_policy Foo bar

vs

> Invalid image_pull_policy 'Foo bar'